source/agora/utils/Test.d

```diff
@@ -707,11 +1015,13 @@ unittest
 {
     import std.range;
 
-    KeyPair[] keys16 = iota(16).map!(_ => KeyPair.random()).array;
-    // Use Genesis
-    KeyPair genesisKP = getGenesisKeyPair();
-    const first = GenesisBlock.txs[0];
-    const resTx1 = first.split([genesisKP], keys16.map!(k => k.address).array);
+    KeyPair[] keys16 = iota(16).map!(idx => WK.Keys[idx]).array;
+    auto first = genesisSpendable.front;
+
+    const resTx1 = TxBuilder(first)
+        .split(keys16.map!(k => k.address).array)
```

Comment:
Since all those unittests do `map` and `array` again, the `map` could be done before.
Alternatively, range overload.